lib/robots/dor_repo/accession/end_accession.rb

```diff
@@ -13,12 +13,7 @@ def perform_work
           next_dissemination_wf = special_dissemination_wf
           workflow_service.create_workflow_by_name(druid, next_dissemination_wf, version: current_version, lane_id:) if next_dissemination_wf.present?
 
-          # Note that this used to be handled by the disseminationWF, which is no longer used.
-          object_client.workspace.cleanup(workflow: 'accessionWF', lane_id:)
-
           start_captioning
-
-          LyberCore::ReturnState.new(status: :noop, note: 'Initiated cleanup API call.')
```

Comment:
this step can now complete immediately instead of waiting for the job to complete it -- this prevents the race conditions